ethereum/referencetests/build.gradle

```
@@ -29,6 +29,37 @@ def blockchainReferenceTests = tasks.register("blockchainReferenceTests")  {
     )
 }
 
+configurations {
+  // we need this because referenceTestImplementation is defined as 'canBeResolved=false'.
+  tarConfig.extendsFrom referenceTestImplementation
+  tarConfig {
+    canBeResolved = true
+    canBeConsumed = false
+  }
+}
+
+def execSpecTests = tasks.register("execSpecTests") {
```

```suggestion
def execSpecTests = tasks.register("executionSpecTests") {
```

I think matching the repo name will make searching slightly easier.